validation/jinja2/validation/advanced_search.html

```diff
@@ -0,0 +1,56 @@
+{#
+    # Displays search form to perform an advanced search
+#}
+   
+{% extends 'validation/_base.html' %}
+{% import 'validation/_macros.html' as macros %}
+
+{% block content %}
+<script>
+    function getSelectedSpeakers() {
+        var items = document.getElementsByName('speaker-options')
+        var selected_speakers = "";
```

Comment:
Two suggestions:
 - suggest using an array instead
 - consistently use `camelCase` in JavaScript, because everybody else does it ¯\\\_(ツ)\_/¯ 

```suggestion
        var selectedSpeakers = [];
```